doc/source/development/env_outside_docker.rst

```diff
@@ -0,0 +1,217 @@
+.. _env_outside_docker:
+
+{{ header }}
+
+===========================================
+Setting up an environment outside of Docker
+===========================================
+
+.. contents:: Table of contents:
+   :local:
+
+Installing a C compiler
+~~~~~~~~~~~~~~~~~~~~~~~
+
+pandas uses C extensions (mostly written using Cython) to speed up certain
+operations. To install pandas from source, you need to compile these C
+extensions, which means you need a C compiler. This process depends on which
+platform you're using.
+
+If you have setup your environment using ``conda``, the packages ``c-compiler``
+and ``cxx-compiler`` will install a fitting compiler for your platform that is
+compatible with the remaining conda packages. On Windows and macOS, you will
+also need to install the SDKs as they have to be distributed separately.
+These packages will be automatically installed by using ``pandas``'s
+``environment.yml``.
```

Comment:
```suggestion
These packages will automatically be installed by using the ``pandas``
``environment.yml`` file.
```
Not mandatory, but slightly better grammar and avoids the hard to pronounce "pandas's"